project/Build.scala

```diff
@@ -2140,6 +2140,10 @@ object Build {
         repl := (`scala3-compiler-bootstrapped` / repl).value,
       ).
       settings(
```

Comment:
maybe this should be in `nonBootstrappedSettings`?

`scala3-bootstrapped/console` will use the snapshot version
```suggestion
      nonBootstrappedSettings(
```

Reply:
imm not actually sure what `nonBootstrappedSettings` does, just saying that this warning only makes sense for console task in non-bootstrapped projects